addons/account/models/chart_template.py

```diff
@@ -785,6 +785,34 @@ def _setup_utility_bank_accounts(self, template_code, company, template_data):
             for company_attr_name, account in zip(accounts_data.keys(), accounts):
                 company[company_attr_name] = account
 
+        # No fields on company
+        is_accounting_installed_next = self.env["ir.module.module"].search([('name', '=', 'accountant')]).state == 'to install'
```

Comment:
The module could also already be installed (i.e. you instantiate the CoA on a new company but all the modules are installed)